### PR TITLE
feat: add essential genes in E. coli

### PR DESCRIPTION
**Pull request by haowang-bioinfo**
### Main improvements in this PR:
* provide the essential genes that are downloaded from the _E. coli_ Genome Project [website](https://www.genome.wisc.edu/resources/essential.htm) from University of Wisconsin-Madison.

**I hereby confirm that I have:**

- [X] Selected `devel` as a target branch

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Ecoli-GEM data
+
+This directory contains datasets that were used for the generation and curation of Ecoli-GEM. The files and their contents are briefly summarized below.
+
+
+- `essentialGenesEcoli.tsv`: The essential genes in E. coli are collected from the _E. coli_ Genome Project [website](https://www.genome.wisc.edu/resources/essential.htm) from University of Wisconsin-Madison.
```
Comment (mihai-sysbio) [inline data/README.md:6]: Style suggestion:

```suggestion
- `essentialGenesEcoli.tsv`: The essential genes in E. coli are collected from the [_E. coli_ Genome Project](https://www.genome.wisc.edu/resources/essential.htm) from University of Wisconsin-Madison (doi:[10.1128/JB.185.19.5673-5684.2003](https://doi.org/10.1128/jb.185.19.5673-5684.2003))
```

**Review by mihai-sysbio — COMMENTED** [review]
Regarding the _Reference_ column, I'm wondering if it shouldn't be switched a more useful PMID or DOI.

**Comment by mihai-sysbio** [comment]
Thanks for the update @haowang-bioinfo, but I was actually referring to the _Reference_ column in the TSV, which at the moment contains very unstructured text. My suggestion would be to replace that with an identifier.

**Comment by mihai-sysbio** [comment]
The previous commit has added some occasional quotes `"` - perhaps one should aim for more consistency. Probably some consensus with `genes.tsv` in Human-GEM would make sense.

**Comment by haowang-bioinfo** [comment]
@mihai-sysbio anything else should be done before merging this?

**Comment by mihai-sysbio** [comment]
Looks good to me @haowang-bioinfo. There is a column `studyType` that is always empty, so if it's not useful now I can also understand if it is removed now and added only once it becomes useful.

**Comment by haowang-bioinfo** [comment]
The column `studyType` is removed

**Review by mihai-sysbio — APPROVED** [review]
🌟

**Pull request merged**